### ISSUE TITLE
Pep8 violation in the code

### ISSUE THREAD

**Issue by Drachenfels** [DRincs-Productions/NQTR-toolkit#101]
In some places, you have syntax like this:

```
if not xalign == None:
```

It's actually against Python coding standard -> https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or

Use either:
```
xalign is None
```
or
```
xalign is not None
```

Statement:
```
not xalign == None
```

Is a double whammy as it's using a double negation, and it's hard to follow when it's True and when it's False, imagine those cases:

```
not "" == None -> True
not None == None -> False
not 0 == None -> True
not 1 == None -> True
not False == None -> True
```

So perhaps simpler:
```
xalign is not None
```
would suffice?

**Comment by BlackRam-oss** [comment]
yes you are right this change should be made, I created a linked branch.
if you want you can do it yourself.

**Comment by Drachenfels** [comment]
Sure thing, will raise PR for you, do you want me to change things in more places as well? Or as little as possible?

**Comment by BlackRam-oss** [comment]
if you want throughout the project. the important thing is that you test that everything works. For dowload the project: https://github.com/DRincs-Productions/NQTR-System?tab=readme-ov-file#to-download-this-test-project

**Comment by Drachenfels** [comment]
Sure thing, I will work on it either tomorrow or Wednesday. Will use black or some other linter to check all the python code.

**Comment by Drachenfels** [comment]
FYI: 

```py
class Button(DisabledClass):
    def __init__(self):
        self.align = None
    
    @property
    def align(self) -> Optional[tuple[Union[int, float], Union[int, float]]]:
        """X align"""
        return self._align
```

Is incorrect.

It works by accident, you change align first to be None and then you change it to be property by side effect of calling this: `self.xalign = 0`

**Comment by Drachenfels** [comment]
I wrote this simple test to showcase it:

```
def test_align_not_working_as_expected():
    from pythonpackages.nqtr.button import Button

    some_button = Button()

    assert some_button.align is None

    some_button.xalign = 0

    assert type(some_button.align) is tuple
    assert some_button.align == (0, 0)
```
